config/default/test/tasks/pregnancy_facility_reminder.spec.js

```diff
@@ -0,0 +1,113 @@
+const chai = require('chai');
+const expect = chai.expect;
+chai.use(require('chai-like'));
+chai.use(require('chai-things'));
+const path = require('path');
+const TestRunner = require('medic-conf-test-harness');
+const { pregnancyRegistrationScenarios, pregnancyHomeVisitScenarios } = require('../form-inputs');
+const { getRangeForTask, getTaskTestDays } = require('../test-helpers');
+const harness = new TestRunner({
+  xformFolderPath: path.join(__dirname, '../../forms/app'),
+});
+
+const now = '2000-01-01';
+const facilityReminderTask = {
+  pre: 3,
+  post: 7,
+  triggers: [14],//2000-01-15, pregnancyRegistrationScenarios.safe
+  triggerUnit: 'days'
+};
+
+const facilityReminderTaskDays = getRangeForTask(facilityReminderTask);
+describe('Pregnancy registration and scheduled visit', () => {
+  before(async () => {
+    return await harness.start();
+  });
+  after(async () => {
+    return await harness.stop();
+  });
+  beforeEach(async () => {
+    await harness.clear();
+    //await harness.flush(1);
+    return await harness.loadForm('pregnancy');
+  });
+  afterEach(() => { expect(harness.consoleErrors).to.be.empty; });
+
+  it('pregnancy registration and scheduled visit', async () => {
+    await harness.setNow(now);
```

Comment:
Typically you'll need to setNow before you load the form.

Reply:
True, moved it to `beforeEach` before loading the form.